jest.config.js

```diff
@@ -82,6 +82,10 @@ module.exports = {
     '<rootDir>/packages/playground/',
     '<rootDir>/samples/'
   ],
+  testRegex: [
```

Comment:
could you explain this? default `testRegex` applies to all `__tests__`  `.js` files, right?